dashboard/v1.1/src/pages/Monitoring/RouteDetails.tsx

```diff
@@ -115,23 +121,47 @@ const RouteDetailsComponent: FC<RouteDetailsProps> = ({
   const data = format(routesChains);
   return (
     <>
-      <span
-        style={{ display: 'flex', alignItems: 'center' }}
-        onClick={() => showDetails(false, routesChains)}
+      <div
+        style={{
+          display: 'flex',
+          justifyContent: 'space-between',
+          alignItems: 'center',
+          position: 'relative'
+        }}
       >
-        <ArrowBackIcon color="primary" fontSize="large" />
-        <span
-          style={{
-            fontSize: '1rem',
-            fontWeight: 'bold',
-            padding: '0 0.4rem',
-            display: 'flex',
-            alignItems: 'center'
-          }}
-        >
-          {truncate(data.name, 70)}
+        <span style={{ display: 'flex', position: 'relative', width: '70%' }}>
+          <ArrowBackIcon
+            color="primary"
+            fontSize="large"
+            style={{ cursor: 'pointer ' }}
+            onClick={() => showDetails(false, routesChains)}
+          />
+          <h3
+            style={{
+              fontSize: '1rem',
+              fontWeight: 'bold',
+              padding: '0.4rem 0.4rem 0 0.4rem',
+              margin: 0,
+              maxWidth: '90%',
+              overflow: 'hidden',
+              textOverflow: 'ellipsis',
+              whiteSpace: 'nowrap'
+            }}
+          >
+            {data.name}
+          </h3>
         </span>
-      </span>
+        <div>
+          <h6 style={{ margin: 0, fontWeight: 'bold' }}>
+            Fetch Time {'    '}: {data.fetchTime.toFixed(3)} ms
+          </h6>
+          {value && data.queryTime[value - 1] ? (
+            <h6 style={{ margin: 0, fontWeight: 'bold' }}>
+              Query Time {'  '}: {data.queryTime[value - 1] / 1000} &micro;s
+            </h6>
+          ) : null}
```

Comment:
2 points:
1. What happens if the time is never in ms or micros? If it's in seconds, then I guess this will show some thousand ms which is never good.
2. I will be happy if this is move concise. A bold query-time is never good. Let's make it compact and cover the timing with a beautify badge that can be green in colour.

Reply:
- I can use the moment library to handle the time symbols. That should solve the issue. May I use that?
- I will surely do the prescribed styling changes.

Reply:
point 1 will be fixed if we follow https://github.com/bench-routes/bench-routes/pull/376#discussion_r598098250.

Reply:
Yeah, using `time.String()` will fix the evaluation time automatically. I was asking about the fetch time, which I am calculating in the frontend because that will give me result in milliseconds : 
```
      const start = performance.now();
      const details = await fetchDetails(instance);
      const end = performance.now();
```

Reply:
Yeah, you should make sure that all units are handled